index.js

```diff
@@ -184,6 +184,90 @@ Dolphin.prototype.docker = function(args){
   });
 }
 
+/**
+ * @param {string | Buffer | string[]} str
+ * @private 
+ */
+function trimTerminator(str) {
```

Comment:
why not just use standard trim? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim

Reply:
Could be ok for the low level cmd call, however for methods like build, where we return the imageId, I think we should perform a regular trim.